README.md

```diff
@@ -1,2 +1,96 @@
 # openproject-plugin_test_action
 A GitHub Action for testing OpenProject plugins
+
```

Comment:
this change can probably just be removed. it comes from the template i initially created the repo with. the readme has nothing to do with my code anymore 🙈